### PR TITLE
logging

### PR DESCRIPTION
**Pull request by erendrake**
We have a lot of different ways of doing logging right now. I Added a static class that will help us have a unified prefix .

### CODE REVIEW

File: InfernalRobotics/InfernalRobotics/Command/Interpolator.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public float GetPosition()
         public void SetIncrementalCommand(float cPosDelta, float cVel)
         {
             float oldCmd = Active ? CmdPosition : Position;
-            //Debug.Log("setIncCmd: oldCmd="+oldCmd.ToString()+", cPosDelta="+cPosDelta.ToString()+",cVel="+cVel.ToString());
```
Comment (erendrake) [inline InfernalRobotics/InfernalRobotics/Command/Interpolator.cs]: if it was commented out i changed it to the superverbose mode that requires you to be in a debug build with the KSP setting for verbose logging turned on.

File: InfernalRobotics/InfernalRobotics/Gui/ControlsGUI.cs [hunk 2]
```diff
@@ -1105,7 +1107,7 @@ private void EditorWindow(int windowID)
                     UpdateGroupEcRequirement(grp);
                     var t = new GUIStyle(GUI.skin.label.name);
                     t.alignment = TextAnchor.MiddleCenter;
-                    GUILayout.Label((string)grp.TotalElectricChargeRequirement.ToString(), t, GUILayout.Width(33), rowHeight);
+                    GUILayout.Label(grp.TotalElectricChargeRequirement.ToString(), t, GUILayout.Width(33), rowHeight);
```
Comment (ZiwKerman) [inline InfernalRobotics/InfernalRobotics/Gui/ControlsGUI.cs]: my compiler on mac was spewing warnings here due to ambiguous overload. but if this compiled for you, should be fine

File: InfernalRobotics/InfernalRobotics/Module/MuMechToggle.cs [hunk 3]
```diff
@@ -960,7 +953,7 @@ protected void UpdatePosition()
             float pos = Interpolator.GetPosition();
             if (rotateJoint)
             {
-                if (rotation != pos) 
+                if (Math.Abs(rotation - pos) > 0.001)
```
Comment (ZiwKerman) [inline InfernalRobotics/InfernalRobotics/Module/MuMechToggle.cs]: don't we need more precision? maybe replace 0.001 with some CONST?

Comment (pellinor0) [inline InfernalRobotics/InfernalRobotics/Module/MuMechToggle.cs]: Good you noted that. I'm aware that comparing floats like this is suspicious, but this wants to know if someone touched the variable, so I'd strongly suggest to stay with "!=".

**Pull request merged**